test/RootStateSchema.json

```diff
@@ -2,6 +2,45 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "additionalProperties": false,
     "definitions": {
+        "ActiveDapp": {
```

Comment:
Changes all seem to be re-aranging or from [this enum to const change](https://github.com/YousefED/typescript-json-schema/pull/535)